packages/gatsby-source-shopify/src/query-builders/locations-query.ts

```diff
@@ -49,12 +49,7 @@ export class LocationsQuery extends BulkQuery {
                 handle
                 id
                 inventoryManagement
-                productBased
```

Comment:
These fields were removed, but `fulfillmentService` is returning `null` from the API so cannot fully test this